src/otx/core/model/module/action_classification.py

```diff
@@ -15,18 +17,26 @@
 from otx.core.model.entity.action_classification import OTXActionClsModel
 from otx.core.model.module.base import OTXLitModule
 
+if TYPE_CHECKING:
+    from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
+
 
 class OTXActionClsLitModule(OTXLitModule):
     """Base class for the lightning module used in OTX detection task."""
 
     def __init__(
         self,
         otx_model: OTXActionClsModel,
-        optimizer: torch.optim.Optimizer,
-        scheduler: torch.optim.lr_scheduler.LRScheduler,
         torch_compile: bool,
+        optimizer: OptimizerCallable = lambda p: torch.optim.SGD(p, lr=0.01),
+        scheduler: LRSchedulerCallable = torch.optim.lr_scheduler.ConstantLR,
```

Comment:
Is the purpose of this to enable the auto-config?

Reply:
I was referring to the way Lightning provides an optimizer and scheduler.
https://lightning.ai/docs/pytorch/stable/cli/lightning_cli_advanced_3.html
It also uses `OptimizerCallable`, `LRSchedulerCallable` to cover all the optimizers and schedulers available in ligthning while making it easy to instantiate.